UPGRADE.md

```diff
@@ -54,6 +54,11 @@ That's why we decided to discontinue TrafficPermission support on the Prometheus
 We support `v1` resources and `v1.0.0` of `gateway-api`. `v1beta1` resources are
 still supported but support for these WILL be removed in a future release.
 
+### KDS Delta enabled by default
+
+KDS Delta is enabled by default. You can fallback to SOTW KDS by setting `KUMA_EXPERIMENTAL_KDS_DELTA_ENABLED=false`.
```

Comment:
This feels a bit confusing. If it's not experimental now (I guess that's why we're setting it as default) the flag should be removed/renamed to:

`KUMA_KDS_DELTA_ENABLED` and set to `true` by default.

The same way we did it for `transparent-proxy` - https://github.com/kumahq/kuma/pull/6083/files#diff-f5545ba9479927ba075dea0dc9b17e5680a7806c7860a43746cbe46d287c36ceL5205-R5244 (we removed `--experimental-transparent-proxy-engine` and added `--use-transparent-proxy-engine-v1`)

Reply:
closing since this is going to be resolved in https://github.com/kumahq/kuma/issues/8290